SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -28,7 +26,7 @@ enum class ERPCType : uint8
 
 	// Helpers to iterate RPC types with ring buffers
 	RingBufferTypeBegin = ClientReliable,
-	RingBufferTypeEnd = NetMulticast
+	RingBufferTypeEnd = CrossServer
```

Comment:
Did this change fix a bug?

Reply:
Nah, but it was necessary to make the way I wrote the StringToRPCType function.